### PR TITLE
Python 3 and PyPy support

### PR DESCRIPTION
**Pull request by jamadden**
- 100% test coverage
- Rename README/CHANGES.txt -> .rst for rendering on github
- Add tox.ini
- Update travis.yml with sudo: false and pip caching and coveralls

### CODE REVIEW

File: src/zope/app/principalannotation/tests.py [hunk 1]
```diff
@@ -13,36 +13,125 @@
...
+
+from zope.traversing.interfaces import ITraverser
+from zope.container.interfaces import INameChooser
+
+class TestImports(unittest.TestCase):
+
+
+    def test_bbb_imports(self):
+        # The most of functionality was moved to zope.principalannotation.
+        # Let's test if old imports still work::
```
Comment (icemac) [inline src/zope/app/principalannotation/tests.py]: The second colon does not seem to be necessary. ;)

**Review by icemac — APPROVED** [review]
LGTM.

**Comment by icemac** [comment]
@jamadden I gave you PyPI rights, too.

File: src/zope/app/principalannotation/tests.py (hunk 1 above)
Comment (jamadden) [reply src/zope/app/principalannotation/tests.py]: Whoops!

**Comment by jamadden** [comment]
Thanks for the review and the rights! 4.0.0 has been pushed to PyPI.

**Pull request merged**